src/coreComponents/constitutive/fluid/multifluid/blackOil/BlackOilFluid.hpp

```diff
@@ -33,7 +34,7 @@ class BlackOilFluid : public BlackOilFluidBase
 {
 public:
 
-  static constexpr real64 minForPhasePresence = 1e-10;
+  static constexpr real64 minForPhasePresence = MultiFluidConstants::minForSpeciesPresence;
```

Comment:
not sure this shortcut needs to be kept. not sure if it is necessary to have the same litteral value stored in the binary 2 times.